Sources/InstantMock/MockUsable/Date+MockUsable.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Date+MockUsable.swift
+//  InstantMock
+//
+//  Created by Arnaud Barisain-Monrose on 22/04/2020.
+//  Copyright © 2017 pirishd. All rights reserved.
+//
+
```

Comment:
Here add `import Foundation`

Otherwise running the command line `swift test` triggers :
```
error: use of undeclared type 'Date'
```

Reply:
Thanks! Is it because Foundation is automatically imported in Xcode projects?

Reply:
I think yes, the Xcode project is configured to find the `Foundation` framework automatically.